app/controllers/api/v1/sources_controller.rb

```diff
@@ -26,7 +26,7 @@ def check_availability
 
         logger.info("Initiating Source#availability_check [#{{"source_id" => source.id, "topic" => topic}}]")
 
-        if Sources::Api::Messaging.topics.include?(topic)
```

Comment:
Is this behavior than just tech debt that no one is using? ie should it get ✂️ too?

Reply:
Yeah afaik this is the only place we were using the topics method, so it can probably be removed from the manageiq-messaging gem.